nested/glustervm.json

```diff
@@ -0,0 +1,212 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "counter": {
+            "metadata": {
+                "description": "from the copyindex function of calling template"
+            },
+            "type": "int"
+        },
+        "moodleCommon": {
+            "metadata": {
+                "description": "Common Moodle values"
+            },
+            "type": "object"
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2015-05-01-preview",
+            "location": "[resourceGroup().location]",
+            "name": "[variables('nicName')]",
+            "properties": {
+                "ipConfigurations": [
+                    {
+                        "name": "ipcfggfs",
+                        "properties": {
+                            "privateIPAllocationMethod": "Dynamic",
+                            "subnet": {
+                                "id": "[variables('subnetSanRef')]"
+                            }
+                        }
+                    }
+                ]
+            },
+            "tags": {
+                "displayName": "Gluster VM NIC"
+            },
+            "type": "Microsoft.Network/networkInterfaces"
+        },
+        {
+            "apiVersion": "[parameters('moodleCommon').computeApi]",
+            "copy": {
+                "count": "[parameters('moodleCommon').glusterDiskCount]",
+                "name": "[variables('diskLoop')]"
+            },
+            "location": "[resourceGroup().location]",
+            "name": "[concat(variables('vmName'),'-datadisk',copyIndex(1))]",
+            "properties": {
+                "accountType": "Premium_LRS",
+                "creationData": {
+                    "createOption": "Empty"
+                },
+                "diskSizeGB": "[parameters('moodleCommon').glusterDiskSize]"
+            },
+            "tags": {
+                "displayName": "Gluster VM Data Disk"
+            },
+            "type": "Microsoft.Compute/disks"
```

Comment:
remove this and just use implicit disks

Reply:
Thanks for the great info. Modified that way.